src/main/scala/com/waioeka/sbt/Jvm.scala

```diff
@@ -31,20 +31,14 @@ import java.lang.management.ManagementFactory
 import org.apache.commons.lang3.SystemUtils
 import sbt._
 
-case class Jvm(classPath: List[File], envParams: Map[String, String]) {
+case class Jvm(classPath: List[File], envParams: Map[String, String], javaOptions: Seq[String]) {
 
   /** Classpath separator, must be ';' for Windows, otherwise : */
   private val sep = if (SystemUtils.IS_OS_WINDOWS) ";" else ":"
 
-  /** Get the JVM options passed into SBT. */
-
-  import scala.collection.JavaConverters._
-
-  private val runtimeArgs = ManagementFactory.getRuntimeMXBean.getInputArguments.asScala.toVector

```

Comment:
Every time a new JVM is spawned from the SBT build, I noticed that the vast majority of SBT plugins out there are allowing users to specify their preferred `javaOptions` so that they don't have to be exactly the same as those the SBT process is running with.